pkg/visibility/api/rest/running_workloads_cq.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rest
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"sigs.k8s.io/kueue/apis/visibility/v1alpha1"
+	"sigs.k8s.io/kueue/pkg/cache"
+	"sigs.k8s.io/kueue/pkg/constants"
+
+	_ "k8s.io/metrics/pkg/apis/metrics/install"
+)
+
+type runningWorkloadsInCqREST struct {
+	c   *cache.Cache
+	log logr.Logger
+}
+
+var _ rest.Storage = &runningWorkloadsInCqREST{}
+var _ rest.GetterWithOptions = &runningWorkloadsInCqREST{}
+var _ rest.Scoper = &runningWorkloadsInCqREST{}
+
+func NewRunningWorkloadsInCqREST(c *cache.Cache) *runningWorkloadsInCqREST {
+	return &runningWorkloadsInCqREST{
+		c:   c,
+		log: ctrl.Log.WithName("running-workload-in-cq"),
+	}
+}
+
+// New implements rest.Storage interface
+func (m *runningWorkloadsInCqREST) New() runtime.Object {
+	return &v1alpha1.RunningWorkloadsSummary{}
+}
+
+// Destroy implements rest.Storage interface
+func (m *runningWorkloadsInCqREST) Destroy() {}
+
+var (
+	debugLog = ctrl.Log.WithName("visibility-server")
+)
```

Comment:
Could we have one logger in this file? Use this one and delete the logger from the `runningWorkloadsInCqREST` struct or the other way around please. If you decide to stick with this one, please rename it to `log` for consitency

Reply:
This variable was used by me during the feature development, and I have removed it as it's no longer necessary.